src/components/apps/toolbar/Toolbar.js

```diff
@@ -121,6 +124,18 @@ function AppsToolbar(props) {
                     selectedCategory={selectedCategory}
                     handleAppNavError={handleAppNavError}
                 />
+                {viewAllApps && (
+                    <Button
+                        id={build(appsToolbarId, ids.VIEW_ALL_APPS)}
+                        variant="outlined"
+                        disableElevation
+                        color="primary"
+                        onClick={viewAllApps}
+                        startIcon={<FilterListIcon />}
+                    >
+                        {getMessage("viewAllApps")}
+                    </Button>
+                )}
                 <div className={classes.divider} />
```

Comment:
If the user is viewing an individual app listing from Search, then they can click on this button to view all apps from there.